### ISSUE TITLE
Большой кэш музыки приводит к вылету втлайт

### ISSUE THREAD

**Issue by Kilkafk** [vtosters/lite#1638]
**Опишите сам баг**
Большой кэш музыки приводит к вылету втлайт

**Опишите действия для воспроизведения ошибки**
1. Выключила интернет
2. Зашла в музыку
3. Жду погрузки
4. Вылетает приложение 

**Опишите ожидаемый результат**
2000 треков должны прогружаться 

![about_device](https://github.com/vtosters/lite/assets/175211713/e67da7fc-0c12-42fd-88fc-0925031e94cc)

**Comment by Wperdex** [comment]
![Screenshot_20240710_142857_VTLite](https://github.com/vtosters/lite/assets/102802937/1a2cd13b-a76e-4e19-8375-6d13b31273fd)

![Screenshot_20240710_142918_VTLite](https://github.com/vtosters/lite/assets/102802937/5ea9fb4c-37b8-4117-be77-46d8ce1f33fe)

**Comment by Kilkafk** [comment]
**Информация об устройстве:** 

- Commit: 93edbf2
- Full Version Information: main-93edbf2-dirty
- Android SDK: 33
- Product: RMX3195RU
- Device: RMX3195
- Board: RM6769
- Manufacturer: realme
- Brand: realme
- Model: RMX3195

**Comment by Kilkafk** [comment]
> В уведомлении появляются логи?

нет, просто вылетает приложение, без ошибок и логов, как будто и не открывала его

**Comment by Kilkafk** [comment]
сейчас в кэше прогружается только кусок песен, не вылетает ничего, но этот кусок начинается не с недавно добавленных, а просто с каких-то, которые я годназад добавила и далее до "ошибки загрузки". при кэшировании большего кол-ва треков - вылетает ошибка "не удалось загрузить аудиозапись"

**Comment by sunmisc** [comment]
я особо не смотрел, но насколько я понимаю данная реализация загружает целиком весь плейлист не используя пагинацию. Ну и загрузка происходит sync. Я стараюсь разобраться с этим, но исходники вк даются тяжело из-за декомпиляции. Возможны и другие мелкие оптимизации

**Comment by WeSTManCoder** [comment]
Присоединяюсь к проблеме. Бывает, что может прогрузиться плейлист, но потом постоянно телефон хочет его закрыть, что он не отвечает, даже если плейлист загрузился, то все равно переодически телефон хочет закрыть приложение.
Телефон: Xiaomi POCO M4 Pro 5G

**Comment by Serdobine** [comment]
@WeSTManCoder а ты как хотел? любому приложению тяжело, когда кэша много.
